RELEASES.md

```diff
@@ -1,3 +1,14 @@
+Version 1.29.2 (2018-10-11)
+===========================
+
+Compatibility Notes
```

Comment:
Is this the right category for these fixes? They don't look like they break backwards compatibility.

Reply:
Other point releases did put all of the fixes in the compatibility notes. I just removed the section name.